typescript/ws-protocol/README.md

```diff
@@ -16,28 +14,13 @@ This package does not require a specific WebSocket server or client implementati
 $ npm install ws
 ```
 
-## Example servers
-
-### System monitor
-
-A system monitor script is provided as an illustrative example of a WebSocket server. To try it out:
-
-1. Run `npx @foxglove/ws-protocol-examples sysmon`.
-2. In a browser, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
-
-<img width="500" alt="Foxglove Studio displaying memory and CPU usage from the system monitor example" src="https://user-images.githubusercontent.com/14237/145313065-85c05645-6b29-4eb2-a498-849c83f8792d.png">
+## Example servers and client
 
-### Generic server template
+Run these [example servers and client](/typescript/ws-protocol-examples), implemented in JavaScript/TypeScript, to get started.
```

Comment:
I think we might want to stick with a full absolute url here, since this readme will be copied to npm where the relative path to another package in the monorepo probably doesn't work